database/migrations/2021_09_12_220452_add_solution_selector_id_to_threads_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddSolutionSelectorIdToThreadsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('threads', function (Blueprint $table) {
+            $table->integer('solution_selector_id')->unsigned()->nullable()->default(null);
```

Comment:
Hi, Nice PullRequest 👏

Why u don’t use `foreignId()` method ? 
[Document](https://laravel.com/docs/8.x/migrations#column-method-foreignId)

thanks.

Reply:
Thank u, but we need `BIGINT` Id!

Reply:
Thanks for the reply.

`foreignId()` generates an `UNSIGNED BIGINT` column, does that mean it has to be `BIGINT` ?
I would like to know the difference between using BIGINT and UNSIGNED BIGINT please :)

Reply:
In this case we have users id of type `int` so the foreign key have to be the same type.
`UNSIGNED` means don't accept negative values.

> This is not the right place for asking questions, please ask in a forum or send me privately.

Thank you.

Reply:
Oh NICE Thanks 👍